Sources/NIO/ChannelHandlers.swift

```diff
@@ -17,6 +17,80 @@
 
 import struct Dispatch.DispatchTime
 
+
+/// `ChannelHandler` which implements some sort of backoff for a `ServerChannel` once `accept` will produce an `IOError`. This is useful as often these errors are recoverable in the sense that we can recover if we slow down the accept rate a bit.
+public final class AcceptBackoffHandler: ChannelDuplexHandler {
+    public typealias InboundIn = Channel
+    public typealias OutboundIn = Channel
+    
+    private var currentBackoffTime: Int?
```

Comment:
this looks like it is an absolute time in nano seconds since boot of the machine, maybe we could improve the name? `nextReadAbsoluteTimeNS` isn't pretty but maybe more clear?